demo/components/FormDemo.js

```diff
@@ -38,7 +38,7 @@ class FormDemo extends React.Component {
         }}>
         <Input
           name='input'
-          label='Input'
+          label={<span>Label <small>(with a label)</small></span>}
```

Comment:
Note: I committed this example from the screenshot. If it seems like a good idea to merge the PropType change then I'd probably want to leave this line here out of the PR.